tests/__init__.py

```diff
@@ -0,0 +1,62 @@
+import os
+import sys
+import tempfile
+import clkhash
+
+TESTDATA = os.path.join(
+    os.path.dirname(__file__),
+    'testdata'
+)
+
+SIMPLE_SCHEMA_PATH = os.path.join(TESTDATA, 'simple-schema.json')
+
+SAMPLE_DATA_SCHEMA_PATH = os.path.join(TESTDATA, 'dirty-data-schema.json')
+
+GOOD_SCHEMA_V1_PATH = os.path.join(TESTDATA, 'good-schema-v1.json')
+GOOD_SCHEMA_V2_PATH = os.path.join(TESTDATA, 'good-schema-v2.json')
+GOOD_SCHEMA_V3_PATH = os.path.join(TESTDATA, 'good-schema-v3.json')
+BAD_SCHEMA_V1_PATH = os.path.join(TESTDATA, 'bad-schema-v1.json')
+BAD_SCHEMA_V2_PATH = os.path.join(TESTDATA, 'bad-schema-v2.json')
+BAD_SCHEMA_V3_PATH = os.path.join(TESTDATA, 'bad-schema-v3.json')
+
+RANDOMNAMES_SCHEMA_PATH = os.path.join(
+    TESTDATA,
+    'randomnames-schema-v2.json'
+)
+
+SAMPLE_DATA_PATH_1 = os.path.join(TESTDATA, 'dirty_1000_50_1.csv')
+SAMPLE_DATA_PATH_2 = os.path.join(TESTDATA, 'dirty_1000_50_2.csv')
```

Comment:
both the SAMPLE_DATA_PATH_i are not used anywhere.

Reply:
We need them when testing good schema